src/pylibsshext/channel.pyx

```diff
@@ -25,7 +25,52 @@ from libc.string cimport memcpy, memset
 
 from pylibsshext.session cimport get_libssh_session
 from pylibsshext.errors cimport LibsshChannelException
-from subprocess import CalledProcessError, CompletedProcess
+from subprocess import CalledProcessError
+
+
+# Defined CompletedProcess here to be compliant with py2
+class CompletedProcess(object):
```

Comment:
Also, I'd probably put it into some `_compat` module instead. It doesn't really belong here.

Besides, I'd use the following style:
```python
try:
    from subprocess import CompletedProcess
except ImportError:
    class CompletedProcess:
        ...
```